src/main/java/Duke.java

```diff
@@ -1,10 +1,155 @@
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedReader;
+
 public class Duke {
+    private static final ArrayList<Task> list = new ArrayList<>();
+
+    private static void update(Task t) {
+        System.out.println("Got it. I've added this task:\n  "
+                + t.toString()
+                + "\nNow you have " + list.size() + " tasks in the list.");
+    }
+
+    private static void delete(int number) {
+        Task toDelete = list.get(number - 1);
+        list.remove(number - 1);
+        System.out.println("Noted. I've removed this task:\n  "
+                + toDelete.toString()
+                + "\nNow you have " + list.size() + " tasks in the list.");
+    }
+
+    private static void readAll() {
+        File f = new File("./Data/");
+        File f2 = new File("./Data/Duke.txt");
+        if (!f.exists()) {
+            f.mkdir();
+        }
+
+        try {
+            f2.createNewFile();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        try {
+            BufferedReader b = new BufferedReader(new FileReader(f2));
```

Comment:
Variable `b` seems to have a large enough scope - it could have a longer, more descriptive name? I saw a similar naming for the `Scanner` variable  `s` below as well.